### DISCUSSION TITLE
syslog ingestion stopped post 2.3.90 upgrade

### DISCUSSION THREAD

**Discussion by FriedrichKayak** [Security-Onion-Solutions/securityonion#6582] [Unsupported Versions]
My previous discussion (#6565) was resolved by creating a new user and marked as answered, which somehow forced everything to work again. However, in another soup upgrade (from 2.3.70 to 2.3.90) related issue, syslog entries are no longer being ingested. I am opening a separate discussion for searchability and because I cannot guarantee these problems have the same root cause.

- Observe using Kibana that for `event.dataset:syslog` events were not ingested after upgrade occurred, but were ingested properly previously.
- Confirm through packet capture that syslog source (in this case a firewall) is blasting packets over to the IP, and the IP is reachable.
- Confirm `so-allow` is still configured properly to allow traffic from firewall.
- Execute `so-filebeat-restart` and `so-logstash-restart`
- Observe `so-status` and all services green and OK.

**Comment by FriedrichKayak** [comment]
Output of the latest `/opt/so/log/filebeat/filebeat.log`:


> 2021-12-15T19:20:37.066Z        WARN    [cfgwarn]       tlscommon/config.go:100 DEPRECATED: Treating the CommonName field on X.509 certificates as a host name when no Subject Alternative Names are present is going to be removed. Please update your certificates if needed. Will be removed in version: 8.0.0
> 2021-12-15T19:20:37.070Z        WARN    beater/filebeat.go:178  Filebeat is unable to load the Ingest Node pipelines for the configured modules because the Elasticsearch output is not configured/enabled. If you have already loaded the Ingest Node pipelines or are using Logstash pipelines, you can ignore this warning.
> 2021-12-15T19:20:37.525Z        WARN    beater/filebeat.go:381  Filebeat is unable to load the Ingest Node pipelines for the configured modules because the Elasticsearch output is not configured/enabled. If you have already loaded the Ingest Node pipelines or are using Logstash pipelines, you can ignore this warning.
> 2021-12-15T19:20:37.654Z        WARN    [cfgwarn]       registered_domain/registered_domain.go:61       BETA: The registered_domain processor is beta.
> 2021-12-15T19:20:37.654Z        WARN    [cfgwarn]       registered_domain/registered_domain.go:61       BETA: The registered_domain processor is beta.
> 2021-12-15T19:20:37.654Z        WARN    [cfgwarn]       registered_domain/registered_domain.go:61       BETA: The registered_domain processor is beta.
> 2021-12-15T19:20:37.654Z        WARN    [cfgwarn]       registered_domain/registered_domain.go:61       BETA: The registered_domain processor is beta.
> 2021-12-15T19:20:37.654Z        WARN    [cfgwarn]       registered_domain/registered_domain.go:61       BETA: The registered_domain processor is beta.
> 2021-12-15T19:20:37.654Z        WARN    [cfgwarn]       registered_domain/registered_domain.go:61       BETA: The registered_domain processor is beta.
> 2021-12-15T19:21:21.597Z        ERROR   [logstash]      logstash/async.go:280   Failed to publish events caused by: EOF
> 2021-12-15T19:21:21.599Z        ERROR   [logstash]      logstash/async.go:280   Failed to publish events caused by: client is not connected
> 2021-12-15T19:21:22.712Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:180  failed to publish events: client is not connected
> 2021-12-15T19:21:25.678Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://so-node-01:5644)): read tcp 172.17.0.22:45710->10.1.10.55:5644: read: connection reset by peer
> 2021-12-15T19:21:33.630Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://so-node-01:5644)): read tcp 172.17.0.22:45754->10.1.10.55:5644: read: connection reset by peer
> 2021-12-15T19:21:43.715Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://so-node-01:5644)): dial tcp 10.1.10.55:5644: connect: connection refused
> 2021-12-15T19:22:09.752Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://so-node-01:5644)): dial tcp 10.1.10.55:5644: connect: connection refused

**Comment by dougburks** [comment]
Have you checked the Elasticsearch and Logstash logs for any clues?
